third_party/terraform/website/docs/guides/provider_reference.html.markdown

```diff
@@ -72,15 +94,6 @@ same configuration.
 
 ### Quick Reference
 
-* `credentials` - (Optional) Either the path to or the contents of a
```

Comment:
Do we want to remove this?

Reply:
I do. I want to bury this configuration parameter at the bottom of the page. IMHO, it shouldn't be used at all and I have an open proposal to remove it entirely from terraform in the next major release.

Reply:
I believe `credentials` is the most commonly used authentication method for the provider. Depriorisiting it in the middle of the `3.X` chain is probably something we should avoid doing, at least until we've committed to removing the field in `4.0.0` (assuming we do so).

Reply:
I'll leave it in the Quick reference but I'll move it down.

Reply:
fixed